src/authorizations/apis/index.ts

```diff
@@ -29,9 +30,23 @@ export const getAuth0Config = async (
     }
     const response = await fetch(url)
     const data = await response.json()
+
     return data
   } catch (error) {
     console.error(error)
     throw error
   }
 }
+
+export const getConnection = async (email: string): Promise<string> => {
```

Comment:
since we're async awaiting i don't believe we'll be returning a promise. We're also potentially returning an error

```suggestion
export const getConnection = async (email: string): string | Error => {
```

Reply:
It does return promise but I will add error
`Promise<string | Error>`

Reply:
I'm confused. Aren't you awaiting the response? That promise should be resolved if that's the case

Reply:
Turns out i don't know anything. You're totally right Subir, my bad